samples/scaleout/distribution-strategy/sample.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Custom distribution strategy
+summary: Use custom distribution strategy to influence how messages are sent to processor instances
+component: Core
+tags:
+- Distributor
+- Scalability
+related:
+- nservicebus/scalability-and-ha
+- samples/scaleout/senderside
+---
+
+NOTE: This sample applies only to version 6 and above.
+
+[The sender side distribution sample](/samples/scaleout/senderside) shows how clients can distribute messages among a number of server instance. Sometimes the built-in round-robin distribution strategy is not feasible though. This sample shows how to build a custom distribution strategy that takes *weight* (an approximation of processing power) into account when deciding where to route a given message.
```

Comment:
shows how clients can distribute messages among a number of server ~~instance~~ instances